templates/components/news-list/news-list.ui_patterns.yml

```diff
@@ -1,10 +1,6 @@
 news-card:
```

Comment:
@jenbreese @boznik This machine name is not what the rest of it is called. This should be changed everywhere it is being used.

Reply:
I fixed that. I had to run lando drush cim to see the config changes.